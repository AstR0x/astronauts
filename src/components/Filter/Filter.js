import React from 'react';

import styles from './Filter.module.css';

export const Filter = ({ filter, onFilter }) => (
  <input
    value={filter}
    onChange={onFilter}
    className={styles.input}
    placeholder="Фильтр"
  />
);
