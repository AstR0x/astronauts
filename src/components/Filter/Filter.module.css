.input {
  margin: 20px auto;
  padding-left: 10px;
  height: 40px;
  width: 100%;
  border: 1px solid #ccc;
  border-radius: 4px;
  outline: none;
}