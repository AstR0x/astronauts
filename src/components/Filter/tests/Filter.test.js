import React from 'react';
import renderer from 'react-test-renderer';

import { Filter } from 'components/Filter';

describe('component Filter should', () => {
  const component = renderer.create(<Filter filter="Jackson" />);
  const tree = component.toJSON();

  it('correct render', () => {
    expect(tree).toMatchSnapshot();
  });
});
