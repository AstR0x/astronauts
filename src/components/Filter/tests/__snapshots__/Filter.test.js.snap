// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component Filter should correct render 1`] = `
<input
  className="input"
  placeholder="Фильтр"
  value="Jackson"
/>
`;
